data/categories.txt

```diff
@@ -43,4 +43,6 @@ category "outfit"
 	"Engines"
 	"Hand to Hand"
 	"Unique"
+	"Minerals"
 	"Special"
+	"Licenses"
```

Comment:
`# Outfits of these types can be bought and sold via the outfitter.`
Not rally about your pr, but this comment is wrong about licenses, they cannot bet bought in the outfitter